app/models/taxon.rb

```diff
@@ -132,18 +132,24 @@ def set_uuid
   validates_uniqueness_of :name, scope: %i[ancestry is_active],
     unless: lambda {
       # If a complex, allow duplicate sibling name for species...
-      ancestry.blank? || !is_active || complex?
+      # Skip validation only if currently provisional AND not changing to non-provisional
+      skip_for_provisional = provisional && ( !will_save_change_to_provisional? || new_record? )
+      skip_for_provisional || ancestry.blank? || !is_active || complex?
     },
     message: "already used as a child of this taxon's parent"
   validates_uniqueness_of :name, scope: %i[ancestry is_active rank],
-    if: -> { complex? }, # ... but do not allow duplicate sibling complex name
+    if: lambda {
+      complex? && !( provisional && ( !will_save_change_to_provisional? || new_record? ) )
+    }, # ... but do not allow duplicate sibling complex name
     message: "already used as a child complex of this taxon's parent"
   validates :name,
     format: { with: TaxonName::SCIENTIFIC_NAME_FORMAT, message: :bad_format },
-    if: proc {| t | t.new_record? || t.name_changed? }
+    unless: proc {| t | t.provisional && ( !t.will_save_change_to_provisional? || t.new_record? ) },
```

Comment:
I highly recommend some validation of provisional names. There are many character we _don't_ want in names, and I think very few additions we might want to allow for provisional names. I'm OK skipping this specific validation for provisional names, but I think we should add a new validation only for provisional names. Perhaps the only needed addition to name format is allowing numbers within words